static/docs/commands-reference/commit.md

```diff
@@ -162,22 +162,22 @@ DVC cache. But it is instructive to verify that's the case.
 First verification:
 
 ```dvc
-    $ dvc status
-
-    evaluate.dvc:
-        changed deps:
-            modified:           data/features
-            modified:           model.pkl
-    train.dvc:
-        changed outs:
-            not in cache:       model.pkl
+$ dvc status
+
+evaluate.dvc:
+    changed deps:
+        modified:           data/features
+        modified:           model.pkl
+train.dvc:
+    changed outs:
+        not in cache:       model.pkl
 ```
 
 And we can look in the DVC cache to see if the new version of `model.pkl` is
 indeed _not in cache_ as claimed. Look at `train.dvc` first:
 
 ```yaml
```

Comment:
I am not sure if the intentation of yaml files also need to be removed. As far as I know its applicable only for dvc code blocks.

@shcheklein can you confirm about it?